epictrack-api/src/api/resources/staff.py

```diff
@@ -100,6 +99,22 @@ def delete(staff_id):
         return 'Staff successfully deleted', HTTPStatus.OK
 
 
+@cors_preflight('PUT')
+@API.route('/<int:staff_id>/last_active_at', methods=['PUT', 'OPTIONS'])
+class StaffLastActiveAt(Resource):
+    """Endpoint resource to return staff last active time."""
+
+    @staticmethod
+    @cors.crossdomain(origin='*')
+    @auth.require
```

Comment:
usual question. This end point will ever update the whole object. Should this be PATCH instead of PUT

Reply:
done